src/main/java/com/telefonica/iot/cygnus/sinks/OrionKafkaSink.java

```diff
@@ -227,18 +228,53 @@ private String buildTopicName() throws Exception {
                     break;
                 case DMBYSERVICEPATH:
                     if (servicePath.equals("/")) {
-                        throw new CygnusBadConfiguration("Default service path '/' cannot be used with "
-                                + "dm-by-service-path data model");
-                    } // if
-                    
-                    name = Utils.encode(servicePath, true, false);
-                    break;
+                        name = Utils.encode(service, false, true);
+                        break;
+                    } else if (servicePath.startsWith("/")) {
```

Comment:
This can be implemented as:

name = Utils.encode(service, false, true) + "_" + Utils.encode(servicePath, true, false);